test/choosetests.jl

```diff
@@ -32,7 +32,7 @@ function choosetests(choices = [])
         "nullable", "meta", "profile", "libgit2", "docs", "markdown",
         "base64", "serialize", "functors", "misc",
         "enums", "cmdlineargs", "i18n", "workspace", "libdl", "int",
-        "intset", "floatfuncs", "compile"
+        "intset", "floatfuncs", "compile", "libgit2-online"
```

Comment:
yeah, where they currently do `runtests.jl pkg` separately, make that `runtests.jl pkg libgit2-online`